sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -791,20 +792,16 @@ protected final void removeTemporaryFiles(Set<String> knownFiles,
 
     /**
      * The MIME type used in the creation of the output channel (if the file system supports it).
-     *
-     * <p>GCS, for example, supports writing files with Content-Type metadata.
-     *
-     * <p>May be overridden. Default is {@link MimeTypes#TEXT}. See {@link MimeTypes} for other
-     * options.
      */
-    protected String mimeType = MimeTypes.TEXT;
+    private final String mimeType;
```

Comment:
Keep a comment that this is the mimetype of the sink and that it can be overridden by the WritableByteChannelFactory